README.md

```diff
@@ -1,310 +1,320 @@
+
+The [RAPIDS](https://rapids.ai) cuGraph library is a collection of graph analytics that process data found in GPU Dataframe - see [cuDF](https://github.com/rapidsai/cudf).  cuGraph aims at provides a NetworkX-like API that will be familiar to data scientists, so they can now build GPU-accelerated workflows more easily.
+
+ For more project details, see [rapids.ai](https://rapids.ai/).
+
+**NOTE:** For the latest stable [README.md](https://github.com/rapidsai/cudf/blob/master/README.md) ensure you are on the `master` branch.
+
+
+
+
+
+
+
+## Getting cuGraph
+### Intro
+There are 4 ways to get cuGraph :
+1. [Quick start with Docker Demo Repo (Coming Soon)](#quick)
+1. [Conda Installation (Coming Soon)](#conda)
+1. [Pip Installation (Coming Soon)](#pip)
+1. [Build from Source](#source)
+
+
+
+
+
+
+
+
+
+As source is currently the only viable option, we'll be providing you with those instructions first.  We'll update the README when the other, easier options become available.
+
+
+
+### Build from Source {#source}
+
+The following instructions are for developers and contributors to cuGraph OSS development. These instructions are tested on Linux Ubuntu 16.04 & 18.04. Use these instructions to build cuGraph from source and contribute to its development.  Other operating systems may be compatible, but are not currently tested.
+
+The cuGraph package include both a C/C++ CUDA portion and a python portion.  Both libraries need to be installed in order for cuGraph to operate correctly.  
+
+The following instructions are tested on Linux systems.
+
+
+
+#### Prerequisites
+
+Compiler requirement:
+
+* `gcc`     version 5.4+
+* `nvcc`    version 9.2
+* `cmake`   version 3.12
+
+
+
+CUDA requirement:
+
+* CUDA 9.2+
+* NVIDIA driver 396.44+
+* Pascal architecture or better
+
+You can obtain CUDA from [https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
+
+
+
+Since `cmake` will download and build Apache Arrow you may need to install Boost C++ (version 1.58+) before running
+`cmake`:
+
+```bash
+# Install Boost C++ for Ubuntu 16.04/18.04
+$ sudo apt-get install libboost-all-dev
+```
+
+or
+
+```bash
+# Install Boost C++ for Conda
+$ conda install -c conda-forge boost
+```
+
+
+
+#### Build and Install the C/C++ CUDA components
+
+To install cuGraph from source, ensure the dependencies are met and follow the steps below:
+
+1) Clone the repository and submodules
+
+  ```bash
+  # Set the localtion to cuGraph in an environment variable CUGRAPH_HOME 
+  export CUGRAPH_HOME=$(pwd)/cugraph
+
+  # Download the cuGraph repo
+  git clone https://github.com/rapidsai/cugraph.git $CUGRAPH_HOME
+
+  # Next load all the submodules
+  cd $CUGRAPH_HOME
+  git submodule update --init --remote --recursive

```

Comment:
I'm still not sure about --remote when we start branching.  It feels like --remote should be used when you're on the development branch.  We should - assuming we continue to do the thirdparty submodule route - pin the versions on a branch, in which case this command does not need the --remote option.

However, more importantly for version 0.6 - this step is unnecessary.  I removed the only submodule that we were using.  We may add it back in version 0.7.  So my inclination would be to just cut the submodule step from this README for now and add it back in once we need it.

Reply:
I'll update and remove `-remote`

Reply:
Removed